sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -416,7 +416,7 @@ def test_tuple_combine_fn_without_defaults(self):
                   max).with_common_input()).without_defaults())
       assert_that(result, equal_to([(1, 7.0 / 4, 3)]))
 
-  def test_to_list_and_to_dict(self):
```

Comment:
I split these into two tests to avoid overwriting the same variable with different types, and it seemed better to test smaller units, regardless